src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -12,8 +12,20 @@ const createZomeCall = instanceCreateZomeCall(INSTANCE_ID)
 
 const MOCK_DEADLINE = '4019-01-02T03:04:05.678901234+00:00'
 
+/* Creates an array of all transactions that inlcude new counterparties for a provided transaction list and counterparty list */
+export const findnewCounterpartiesFromList = (transactionList = [], counterpartyList = []) => {
```

Comment:
when naming array variables, it's simpler to just name them the a plural. That's all the clue we need to get that it's an array.
So I would rename these variables `transactions` and `counterparties`.